<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipPlacementMenu</class>
 <widget class="QMainWindow" name="ShipPlacementMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Морской бой - Размещение кораблей</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(
    x1: 0, y1: 0,
    x2: 1, y2: 1,
    stop: 0 #a0e9ff,
    stop: 1 #00bcd4
);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>491</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="fieldGrid"/>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>441</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(
        x1: 0, y1: 0,
        x2: 1, y2: 1,
        stop: 0 #a0e9ff,
        stop: 1 #00bcd4
);
border: 3px solid rgba(0, 100, 130, 0.9); 
border-radius: 14px;
padding: 12px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnSize1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 0.85);
    border: 1px solid #007c91;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 1);
}</string>
     </property>
     <property name="text">
      <string>1-палубный</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSize4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 0.85);
    border: 1px solid #007c91;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 1);
}</string>
     </property>
     <property name="text">
      <string>4-палубный</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSize3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 0.85);
    border: 1px solid #007c91;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 1);
}</string>
     </property>
     <property name="text">
      <string>3-палубный</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSize2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 255, 255, 0.85);
    border: 1px solid #007c91;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 1);
}</string>
     </property>
     <property name="text">
      <string>2-палубный</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSize2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #003f5c; 
font-weight: bold;
font-size: 14px;
background-color: rgba(255, 255, 255, 0.6);
padding: 4px 8px;
border-radius: 6px;
border: 1px solid #007c91; </string>
     </property>
     <property name="text">
      <string>Осталось:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSize1">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #003f5c; 
font-weight: bold;
font-size: 14px;
background-color: rgba(255, 255, 255, 0.6);
padding: 4px 8px;
border-radius: 6px;
border: 1px solid #007c91; </string>
     </property>
     <property name="text">
      <string>Осталось:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSize3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #003f5c; 
font-weight: bold;
font-size: 14px;
background-color: rgba(255, 255, 255, 0.6);
padding: 4px 8px;
border-radius: 6px;
border: 1px solid #007c91; </string>
     </property>
     <property name="text">
      <string>Осталось:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSize4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #003f5c; 
font-weight: bold;
font-size: 14px;
background-color: rgba(255, 255, 255, 0.6);
padding: 4px 8px;
border-radius: 6px;
border: 1px solid #007c91; </string>
     </property>
     <property name="text">
      <string>Осталось:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_player">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>20</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;
font-weight: bold;
color: #2C3E50;
padding: 10px;
border-bottom: 2px solid #2980B9;
background-color: #ECF0F1;
qproperty-alignment: 'AlignCenter';
</string>
    </property>
    <property name="text">
     <string>Игрок 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>500</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-image: url(:/images/images/restart_icon.png);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent;
    border: none;
    width: 32px;
    height: 32px;
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-image: url(:/images/images/return_button.png);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent;
    border: none;
    width: 32px;
    height: 32px;
}

QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: transparent;
font-size: 12pt;
font-weight: bold;
padding-top: 4px;
qproperty-alignment: AlignHCenter;</string>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 14pt;
font-weight: bold;
padding-top: 4px;
qproperty-alignment: AlignHCenter;</string>
    </property>
    <property name="text">
     <string>Игрок 1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Морской Бой - Размещение</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Морской Бой - Размещение кораблей</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
